modules/ROOT/pages/request-reply-scope.adoc

```diff
@@ -1,11 +1,11 @@
-= About the Request-Reply Scope
+= Request-Reply Scope
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: request reply, asynchronous
 
-The Request-Reply Scope enables you to embed a pocket of asynchronous processing within a Mule flow. This functionality enables you to receive a response from an asynchronous flow without hardcoding the destination of the response. +
-For example, you can use request-reply to convert a one-way VM or JMS connector flow to a request-response flow without having to change it's configuration. In other words, the request-reply converts part of a synchronous process into an asynchronous one.
+The Request-Reply Scope enables you to embed a pocket of asynchronous processing within a flow. This functionality enables you to receive a response from an asynchronous flow without hardcoding the destination of the response. +
+For example, you can use request-reply to convert a one-way VM or JMS connector flow to a request-response flow without having to change it's configuration. The request-reply converts part of a synchronous process into an asynchronous one.
 
 == Basic Anatomy
```

Comment:
I just spotted this while skimming down to the new text, and I recall seeing this heading in another job, so I am guessing that it appears in more than one doc? 
I can't remember my advice from that first instance, but I think we want to refer to "architecture" rather than "anatomy." You might share with your team if this is something common in the doc set.